modules/monitoring-creating-cluster-id-labels-for-metrics.adoc

```diff
@@ -96,7 +96,7 @@ data:
           targetLabel: cluster_id <2>
           action: replace <3>
 ----
-<1> The system initially applies a temporary cluster ID source label named `+++__tmp_openshift_cluster_id__+++`. This temporary label gets replaced by the cluster ID label name that you specify.
+<1> The system initially applies a temporary cluster ID source label named `+++__tmp_openshift_cluster_id__+++`, its value is set to the one from https://github.com/openshift/openshift-docs/blob/main/modules/support-get-cluster-id.adoc (ADJUST). This temporary label gets replaced by the cluster ID label name that you specify.
```

Comment:
TODO: this still needs some work.

How to include the link? additional resources?

Reply:
@machine424 We're not allowed to include github links in the core OCP docs without special permission from PX and support. But, yes, if we were to include a link, we would put it under Additional resources in the assembly adoc file where this module adoc file is included. Is the link absolutely necessary? Let me think of other options, and I will drop in a comment here with how I think we can solve it without a link.

Reply:
@machine424 Yes. We can add an xref to the Additional resources section in the assembly adoc file in which this adoc file is included. Lmk if you need help with that

Reply:
@machine424 - For the xref link, add the following code at line 186 of `configuring-the-monitoring-stack.adoc` under `.Additional resources` and leave a blank line below it:

`* For information about how to get your cluster ID, see xref:../support/gathering-cluster-data.adoc#support-get-cluster-id_gathering-cluster-data[Obtaining your cluster ID].
`

Reply:
Ok, let's put it under under `.Additional resources`
Do you mean `modules/monitoring-configuring-the-monitoring-stack.adoc`, that file only has ~150 lines on main.

Reply:
done.

Reply:
Got `ERROR (monitoring/configuring-the-monitoring-stack.adoc): "xref:../support/gathering-cluster-data.adoc#support-get-cluster-id_gathering-cluster-data[Obtaining your cluster ID]" appears to try to reference a file not included in the "openshift-rosa" distro`
If it'll be easier for you to do it yourself, don't hesitate to close this and take over.